flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/plan/batch/table/validation/SortValidationTest.scala

```diff
@@ -26,22 +26,6 @@ import org.junit._
 
 class SortValidationTest extends TableTestBase {
 
-  @Test(expected = classOf[ValidationException])
```

Comment:
We could activate these tests instead of deleting them?

Reply:
The fetch is activated in the `LimitITCase`. I will add a offset test as well there.